### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

**Comment by polarker** [comment]
@Lbqds I need your help to fix the tests. I don't have the time for it.

**Review by Lbqds — APPROVED** [review]
LGTM, thanks!

File: alephium/contracts/token_bridge/token_bridge_for_chain.ral [hunk 1]
```diff
@@ -62,7 +62,6 @@ Contract TokenBridgeForChain(
         for (let mut index = 0; index < length; index = index + 8) {
             let undoneSequenceId = subContractId!(byteVecSlice!(paths, index, index + 8))
             UndoneSequence(undoneSequenceId).destroy()
-            index = index + 8
```
Comment (polarker) [inline alephium/contracts/token_bridge/token_bridge_for_chain.ral:66]: Thanks!

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

**Comment by polarker** [comment]
I have done my last big round of review, all looks great to me now. There are two more minor ideas:
1. We could decompose `token_bridge.ral` into an abstract contract `token_bridge_governance.ral` and `token_bridge.ral` which inherits the abstract governance contract. This would be similar to Ethereum and probably help code review.
2. We could add a test for public functions for the main contracts like `token_bridge.ral`, so we make sure that we will not break the visibilities of the functions.

I will also spend sometime to think about how to improve WrappedAlph and refundAddress.

**Pull request merged**